### PR TITLE
Backport #14219 to branch/v10

### PR DESCRIPTION
**Pull request by russjones**
Backport #14219 to branch/v10

### CODE REVIEW

**Review by timothyb89 — APPROVED** [review]

File: docs/pages/faq.mdx [hunk 1]
```diff
@@ -142,6 +142,18 @@ Teleport provides security-critical support for the current and two previous rel
 See our [Upgrading](./management/operations/upgrading.mdx) guide for more
 information.
 
+## Which operating systems are supported?
+
+Teleport binaries runs on Linux, macOS, and Windows.
```
Comment (r0mant) [inline docs/pages/faq.mdx:147]:
```suggestion
Teleport binaries run on Linux, macOS, and Windows.
```

File: docs/pages/faq.mdx [hunk 2]
```diff
@@ -142,6 +142,18 @@ Teleport provides security-critical support for the current and two previous rel
...
+
+|Operating System| Version                 |
+|----------------|-------------------------|
+| macOS          | 10.13+ (High Sierra)    |
+| Windows        | Windows 10+ (rev. 1607) |
```
Comment (r0mant) [inline docs/pages/faq.mdx:152]: Should we mention this is client-only?

**Comment by zmb3** [comment]
Feels like this is somewhat duplicative of the OS support section on the installing page: https://goteleport.com/docs/installation/#operating-system-support

Can we unify these?

**Comment by russjones** [comment]
Closing this in-favor of https://github.com/gravitational/teleport/pull/16780.

**Pull request closed without merging**